hack/tools.go

```diff
@@ -0,0 +1,22 @@
+// +build tools
+
+/*
+Copyright 2020 Google LLC.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package tools imports tool dependencies
+package tools
+
+import _ "knative.dev/pkg/hack"
```

Comment:
Please add the imports for https://github.com/google/knative-gcp/blob/master/Gopkg.toml#L7-L11 and https://github.com/google/knative-gcp/blob/master/Gopkg.toml#L20. These are required for testing.